language/move-bytecode-verifier/bytecode-verifier-tests/src/unit_tests/catch_unwind.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) The Move Contributors
```

Comment:
this is great thanks, though in time it would be good to have some multithread tests with injected panics. 
Also it would be lovely to have a OOM error and see what happens.
Again up to you and we can do that later